python-lib/dku_utils/taints.py

```diff
@@ -0,0 +1,80 @@
+import json, logging
+from dku_utils.access import _is_none_or_blank
+
+class Taint(dict):
+    def __init__(self, taint):
+        logging.debug("Creating taint from '%s'" % json.dumps(taint))
+        if isinstance(taint, str):
```

Comment:
should we check both `str` and `basestring` for Python2 BW compat?

Reply:
We dropped python 2 support for AKS recently. I try to add back compatibility for existing code envs that might not be supported, but I think if you update the plugin, it'll ask you to create a new code env.